CharlieBackend.Api/Controllers/SchedulesController.cs

```diff
@@ -20,6 +20,9 @@ namespace CharlieBackend.Api.Controllers
     [ApiController]
     public class SchedulesController : ControllerBase
     {
+        /// <summary>
+        /// 
+        /// </summary>
         public readonly IScheduleService _scheduleService;
```

Comment:
you can make it private, so VS will not request to commet private field, against mocks